researcher_ids/RI_Claims_V1.md

```diff
@@ -60,90 +65,85 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
--   **[optional]** : A field that MAY be present within a
-    [Claim Object](#claim-object-fields).
--   **[optional on specific RI claims]** : A field that MAY be present only on
-    claims when where the claim definition explicitly mentions that this field
-    MAY be specified. All other standard GA4GH RI claims MUST NOT define this
-    field. This field MAY be used on
-    [custom claims](#custom-researcher-identity-claim-names) and it is
-    RECOMMENDED for the page contents within the custom claim name URL to
-    define whether this field may be present or not.
-
 #### **Researcher Identity Claim ("RI Claim")**
 
--   A set of [RI Claim Objects](#researcher-identity-claim-object-ri-claim-object)
+-   A set of [RI Claim Embedded Tokens](#researcher-identity-claim-embedded-token-ri-claim-embedded-token)
     provided by a common key value within the "ga4gh" OIDC claim. For example, the
     following structure encodes a "ga4gh.ControlledAccessGrants" RI Claim:
 
     ```
     "ga4gh" : {
       "ControlledAccessGrants": [
-        Claim Object (see definition below),
-        Claim Object (if more than one),
+        <RI Claim Embedded Token (see definition below)>,
+        <RI Claim Embedded Token (if more than one)>,
         ...
       ]
     }
     ```
 
     RI Claims can be bundled together in a [Passport](#passport).
 
-#### **Researcher Identity Claim Object ("RI Claim Object")**
+#### **Researcher Identity Claim Embedded Token ("RI Claim Embedded Token")**
 
--   An Assertion from [Claim Authority](#claim-authority) that is bound to a
-    researcher identity.
+-   An Assertion from a [Claim Source](#claim-source) that is bound to a
+    researcher identity and signed by an
+    [Embedded Claim Signatory](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-claim-signatory)
+    as per the "iss" field.
 
--   Encoded as a JSON object that contains various properties
-    ([Claim Object Fields](#claim-object-fields)) that describe the assertion and
+-   Encoded as an
+    [Embedded Token](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-token)
+    that contains various properties
+    ([RI Claim Object Fields](#ri-claim-object-fields)) that describe the assertion and
     limitations thereof.
+    
+-   RI Claim Embedded Tokens are encoded as strings and included as a list of
+    strings returned within a [RI Claim](#researcher-identity-claim-ri-claim). See
+    the [RI Claim Embedded Token](#ri-claim-embedded-token) section of this
+    specification for more details.
 
-#### **Claim Authority**
+#### **Researcher Identity Claim Object ("RI Claim Object")**
 
--   The
-    [AAI Claim Authority](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-claim-authority)
-    as encoded by the [source field](#source-required).
+An OIDC claim "ga4gh_rio" that contains fields that provide GA4GH
```

Comment:
The ga4gh_rio claim **is not**  an OIDC claim. It is a JWT claim as specified in the section 2 "Terminology" of RFC 7519 "JSON Web Token", see https://tools.ietf.org/html/rfc7519#section-2
OIDC claims as defined in https://openid.net/specs/openid-connect-core-1_0.html#Claims are defined as members of JSON object returned by /userinfo or in id_token.

Thus this section should be reworded so that "OIDC" should be replaced with "JWT", and it should mention the 
_JSON Web Token Claims Registry_ 
https://tools.ietf.org/html/rfc7519#section-10.1 instead of OIDC specification.

Reply:
Done